utils/validate-install-plan.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { fetchPaginatedGHResults } = require('./github-api-helpers');
+const { findMainInstallConfigFiles, readQuickstartFile } = require('./helpers');
+const path = require('path');
```

Comment:
Could we move this to the top of the `require` lines? Looks like most of these modules follow an order of: 

_external
internal_